src/test/java/core/service.impl/ParserImplTest.java

```diff
@@ -0,0 +1,66 @@
+package core.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.models.FruitTransaction;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.impl.ParseImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class ParserImplTest {
+
+    public void expectedList(List<FruitTransaction> fruitTransactionsList) {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private List<FruitTransaction> getExpectedTransactions() {
```